packages/ua-devtools-solana/src/oft/config.ts

```diff
@@ -43,72 +43,37 @@ const onlyEdgesFromSolana = (
     }
 }
 
-export const initNonce: OAppConfigurator = createConfigureEdges(
-    onlyEdgesFromSolana(async ({ vector: { to } }, sdk: OFT) => {
-        const logger = createOFTLogger()
-
-        if (typeof sdk.initializeNonce !== 'function') {
-            return logger.warn(`Could not find initializeNonce() method on OFT SDK, skipping`), undefined
-        }
-
-        if (typeof sdk.isNonceInitialized !== 'function') {
-            return logger.warn(`Could not find isNonceInitialized() method on OFT SDK, skipping`), undefined
-        }
-
-        const isInitialized = await sdk.isNonceInitialized(to.eid, to.address)
-        return isInitialized ? undefined : sdk.initializeNonce(to.eid, to.address)
-    })
-)
-
-export const initSendLibrary: OAppConfigurator = createConfigureEdges(
+export const initConfig: OAppConfigurator = createConfigureEdges(
     onlyEdgesFromSolana(async ({ vector: { to } }, sdk) => {
-        const logger = createOFTLogger()
-
-        if (typeof sdk.initializeSendLibrary !== 'function') {
-            return logger.warn(`Could not find initializeSendLibrary() method on OFT SDK, skipping`), undefined
-        }
-
-        if (typeof sdk.isSendLibraryInitialized !== 'function') {
-            return logger.warn(`Could not find isSendLibraryInitialized() method on OFT SDK, skipping`), undefined
+        if (typeof sdk.initConfig !== 'function') {
+            return createOFTLogger().warn(`Could not find initConfig() method on OFT SDK, skipping`), undefined
         }
-
-        const isInitialized = await sdk.isSendLibraryInitialized(to.eid)
-        return isInitialized ? undefined : sdk.initializeSendLibrary(to.eid)
+        return sdk.initConfig(to.eid)
     })
 )
 
-export const initReceiveLibrary: OAppConfigurator = createConfigureEdges(
-    onlyEdgesFromSolana(async ({ vector: { to } }, sdk) => {
+export const addRemote: OAppConfigurator = createConfigureEdges(
```

Comment:
See below - this should also be placed in `setPeer` on the SDK I think

Reply:
Tested and done.